geode-core/src/main/java/org/apache/geode/distributed/internal/AbstractDistributionConfig.java

```diff
@@ -1312,11 +1313,13 @@ static Class _getAttributeType(String attName) {
     m.put(MEMCACHED_BIND_ADDRESS,
         "The address the GemFireMemcachedServer will listen on for remote connections. Default is \"\" which causes the GemFireMemcachedServer to listen on the host's default address. This property is ignored if memcached-port is \"0\".");
     m.put(REDIS_PORT,
-        "The port GeodeRedisServer will listen on. Default is 0. Set to zero to disable GeodeRedisServer.");
+        "The port Redis API for Geode will listen on. Default is 0");
```

Comment:
Is the default now 6379? Also I think you want a "." at the end of the sentence.

Reply:
Yes. Thank you for finding this!